includes/forum-reactions.php

```diff
@@ -154,6 +154,11 @@ public function reaction_change($post_id, $user_id, $reaction) {
             return false;
         }
 
+        // ... and the user is not banned ...
+        if ($this->asgarosforum->permissions->isBanned($user_id)) {
+            return false;
+        }
+
```

Comment:
We should do the same check in the `render_reactions()` function to ensure that no unnecessary reaction-link is generated for banned users.

Reply:
Good point.